stackit/internal/services/objectstorage/objectstorage_acc_test.go

```diff
@@ -221,8 +213,9 @@ func TestAccObjectStorageResource(t *testing.T) {
 					}
 					return fmt.Sprintf("%s,%s,%s", testutil.ProjectId, credentialsGroupId, credentialId), nil
 				},
-				ImportState:       true,
-				ImportStateVerify: true,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"access_key", "secret_access_key"},
```

Comment:
Here only `secret_access_key` is sensitive, right? Why can't we import `access_key`?

Reply:
As far as I know, only `secret_access_key` is sensitive. But `access_key` isn't provided as well in the GET response, so can't be fetched for imported resources